docs/managing-dragonfly/backups.md

```diff
@@ -168,14 +168,15 @@ $> ./dragonfly --logtostderr --dir s3://acme-backups/dragonfly/staging
 
 As with disk backups, Dragonfly will automatically load the latest snapshot from the S3 bucket on startup and store a snapshot on shutdown.
 
-:warning: Dragonfly S3 cloud storage is a preview feature and does not yet support HTTPS. Please open a [GitHub](https://github.com/dragonflydb/dragonfly) issue if you find any problems or require HTTPS support.
+:warning: Dragonfly S3 cloud storage is a preview feature. Please open a [GitHub](https://github.com/dragonflydb/dragonfly) issue if you find any problems.
```

Comment:
Oh I see. Just want to confirm that this is about the direct S3 snapshots, not via the K8s operator.

Reply:
yep s3 snapshots